cg/cli/workflow/rnafusion/base.py

```diff
@@ -42,3 +50,56 @@ def rnafusion(context: click.Context) -> None:
 rnafusion.add_command(store_housekeeper)
 rnafusion.add_command(store)
 rnafusion.add_command(store_available)
+
+
+@rnafusion.command()
+@ARGUMENT_CASE_ID
+@click.pass_obj
+def dev_config_case(config: CGConfig, case_id: str):
+    """Configure an RNAFusion case so that it is ready to be run."""
+    factory = ConfiguratorFactory(config)
+    configurator: NextflowConfigurator = cast(
+        NextflowConfigurator, factory.get_configurator(Workflow.RNAFUSION)
+    )
```

Comment:
Is this casting made to make the type hint correct? If that is the case why not do
```suggestion
    configurator: RnaFusionConfigurator = factory.get_configurator(Workflow.RNAFUSION))
```
Or why is this done then?

Reply:
you can resolve this